src/logic/Board.ts

```diff
@@ -20,14 +21,23 @@ export class Board implements IBoard {
     this.grid = new Grid(Tile, spiral({ radius: 4 }));
   }
 
+  /**
+   * returns all neighbour tiles of a coordinate, within grid limits
+   * @param coord a hexcoordinate
```

Comment:
I like the idea, but not sure this is particularly helpful, example values might help. Might also help to agree on standardized terms and use throughout.